CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+If you would like to contribute code to this project you can do so through GitHub by
+forking the repository and sending a pull request.
+
+When submitting code, please make every effort to follow existing conventions
+and style in order to keep the code as readable as possible.
```

Comment:
Do we have a document we can reference to help contributors out here?

(Alternatively, we might want to consider setting up some tooling to help them out, like Prettier, spotless, etc.)

Reply:
Good questions. For spotless, it's [already set](https://github.com/cashapp/kfactories/blob/42a0acf9c6b9c078de92dc6655ea99e0936a3a41/build.gradle#L72-L83). I'll add a link to our Square styleguide.